### PR TITLE
_posts/2019-01-29-yocto-meta-rte-migrate-to-github.md: new article

### PR DESCRIPTION
**Pull request by MartaSzel**

### CODE REVIEW

File: _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    - meta-rte
+    - linux
+    - rte
+categories:
+    - OS Dev
+---
+
+## Yocto meta-rte migrate to GitHub
+
+We are happy to announce that our 3mdeb’s Yocto `meta-rte` is now available on our GitHub.
```
Comment (artur-rs) [inline _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md]: @MartaSzel here should be linked our GitHub url - at this moment this should be ok:  ( \[GitHub\]\(link-TBD\) )
After `meta-rte` migration take place, you should update link

File: _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+but thanks to Yocto Project bringing embedded devices to market becomes easier, cheaper
+and faster. And it doesn't even matter what kind of device it is. In our case, it's 3mdeb's
+[RTE](https://shop.3mdeb.com/product/rte/). If you have any questions about rte, `meta-rte`
+or you are just interested in embedded systems you can email us at: contact@3mdeb.com.
+The confirmation of our competence is our presence on the [Embedded Linux Expert List](https://elinux.org/Experts#The_List)
+and [Yocto Project Consultants List](https://www.yoctoproject.org/community/consultants/).
+
+You should also know that anyone can build a system based on `meta-rte`. To do this you will
+need a tool named `kas`. Interested? Check our another article:
+https://github.com/3mdeb/news-and-ideas/blob/kas/kas.md.
```
Comment (artur-rs) [inline _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md]: this link should be changed for correct one redirecting to our blog and specific post

File: _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md (hunk 1 above)
Comment (artur-rs) [reply _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md]: @MartaSzel https://github.com/3mdeb/meta-rte

File: _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+features of the RTE extension boards:
+  * UART (uart2) interface - for RS-232
+  * I2C (i2c1) interface - for mcp2301 GPIO expander
+  * SPI (spi1) interface - for flashing external boards' SPI flash chips
+  * USB (ehci/ohci 2 and 3) interfaces - for additonal on-board USB connector
+
+## System features:
+
+* minimal image with full support for the target hardware,
+* RteCtrl utility - controlling the RTE via REST API calls,
```
Comment (artur-rs) [inline _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md]: @MartaSzel link to our GitHub: https://github.com/3mdeb/RteCtrl

File: _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  * etc.
+* Utilities for controlling the platform under test via RTE:
+  * ser2net - redirecting platform's serial via Ethernet over telnet
+  * flashrom - flashing platform's SPI chip,
+  * fastboot and imx-usb-loader https://github.com/vitroTV/imx_usb_loader for
+    i.MX6 boards flashing,
+  * stlink and openocd for STM32 microcontrollers flashing
+  * ifdtools, cbftools - utilites useful for coreboot testing
+
+[image]
```
Comment (artur-rs) [inline _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md]: @MartaSzel https://3mdeb.com/wp-content/uploads/2019/02/REST-API.png

File: _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    - meta-rte
+    - linux
+    - rte
+categories:
+    - OS Dev
+---
+
+## Yocto meta-rte is now open for everyone
+
+[image-yocto]
```
Comment (artur-rs) [inline _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md]: @MartaSzel https://3mdeb.com/wp-content/uploads/2019/02/YoctoProject_Logo_RGB.jpg

File: _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+* Utilities for controlling the platform under test via RTE:
+  * ser2net - redirecting platform's serial via Ethernet over telnet
+  * flashrom - flashing platform's SPI chip,
+  * fastboot and imx-usb-loader https://github.com/vitroTV/imx_usb_loader for
+    i.MX6 boards flashing,
+  * stlink and openocd for STM32 microcontrollers flashing
+  * ifdtools, cbftools - utilites useful for coreboot testing
+
+![API](https://3mdeb.com/wp-content/uploads/2019/02/REST-API.png)
+Here we have our dev version API. It starts automatically and thanks to that you can
```
Comment (artur-rs) [inline _posts/2019-01-29-yocto-meta-rte-migrate-to-github.md]: Missing new line after image

**Comment by pietrushnic** [comment]
@artur-rs is this Yoast optimized? If not please fix all Yoast problems and then schedule the post. @macpijan @lukasz-laguna @miczyg1 is there any document with the process of blog post-publication? If not we should try to prepare that with @Moviss

**Comment by miczyg1** [comment]
@pietrushnic I have no clue if we have such document.

**Comment by artur-rs** [comment]
@Moviss you can schedule this blog-post

**Comment by Moviss** [comment]
@artur-rs it is scheduled on march 13 2019.

**Comment by pietrushnic** [comment]
@Moviss thank you.

**Pull request merged**